### PR TITLE
release/v2.0.0

### PR DESCRIPTION
**Pull request by theborch**

### CODE REVIEW

File: src/pybritive/britive_cli.py [hunk 1]
```diff
@@ -168,10 +175,22 @@ def _display_banner(self):
...
 
         # if we get here then we need to at least grab the banner and see if it has changed
-        banner = self.b.banner()
-        banner_changed = Cache().save_banner(tenant=self.tenant_name, banner=banner)
-        if banner and banner_changed:
-            self.print(f'*** {banner.get("messageType", "UNKNOWN")}: {banner.get("message", "<no message>")} ***')
+        if banner := self.b.banner():
+            banner_changed = Cache().save_banner(tenant=self.tenant_name, banner=banner)
+            msg_type = banner.get('messageType', 'UNKNOWN')
+            print(f"{fore.RED}HERE HERE HERE{style.RESET}")
```
Comment (twratl) [inline src/pybritive/britive_cli.py:181]: this would fail for >= Python3.10?

File: src/pybritive/britive_cli.py [hunk 2]
```diff
@@ -201,7 +220,7 @@ def _is_saml_user(token):
             ).get('username', '')
 
             return username.startswith('SAML')
```
Comment (twratl) [inline src/pybritive/britive_cli.py:222]: this is likely changing with Britive Internal Directory. Let's see what updates need to occur.

File: src/pybritive/completers/__init__.py [hunk 3]
```diff
@@ -1,11 +1,2 @@
...
-import pkg_resources
-
-# click < 8.0.0 does shell completion different...
-# not all the classes/decorators are available, so we cannot
-# create custom shell completions like we can with click > 8.0.0
-click_major_version = int(pkg_resources.get_distribution('click').version.split('.')[0])
-if click_major_version >= 8:
-    from . import powershell_completion
-    from . import bash_gte_42
+from . import bash_gte_42, powershell_completion  # noqa: F401
```
Comment (twratl) [inline src/pybritive/completers/__init__.py:2]: not sure we can stop supporting click 7.x, as other CLI tools (Azure CLI for sure) requires click 7.x. can we confirm that python3.8 and up are not supported by click 7.x?

File: src/pybritive/britive_cli.py (hunk 1 above)
Comment (theborch) [reply src/pybritive/britive_cli.py]: removed, leftover print test

**Pull request merged**